axelor-account/src/main/java/com/axelor/apps/account/service/NotificationServiceImpl.java

```diff
@@ -242,4 +248,15 @@ protected MoveLine findInvoiceAccountMoveLine(Invoice invoice) {
     }
     throw new NoSuchElementException();
   }
+
+  protected MoveLine findSubrogationReleaseAccountMoveLine(Invoice invoice) throws AxelorException {
```

Comment:
Consider using java 8 stream and optional to highlight that the returned value can be empty.